Cargo.lock

```diff
@@ -1,36 +1,3 @@
-[root]
```

Comment:
Any idea why the lock file changed?  Wondering if we should bother including it in this PR

Reply:
Oh I though I've excluded it. I don't know why it was edited, basically a section moved, but nothing changed